### DISCUSSION TITLE
Parameter capital letter ?

### DISCUSSION THREAD

**Discussion by CharlotteJu** [simolus3/drift#2546] [Q&A] [answered]
Is it possible to pass a parameter to Drift to have the name of each attribute and data class start with a capital letter automatically ?

**Comment by simolus3** [comment]
I'm not sure I understand what you mean? Do you mean the names in SQL or those in Dart? Drift re-uses the getter names from table classes in Dart, so couldn't you change them to start with an uppercase letter?

**Reply by CharlotteJu** [reply]
Yes, sorry for not being clear. I was referring to the naming convention in SQL. Are there any parameters to indicate that the table or attribute names should start with an uppercase letter in the SQL script? Also, is there a parameter to automatically use PascalCase instead of snake_case (again, for the SQL script)?

**Reply by simolus3** [reply, marked as answer]
Yes, you can use the `case_from_dart_to_sql` [builder option](https://drift.simonbinder.eu/docs/advanced-features/builder_options/), which you can set to `PascalCase` for this.

**Reply by CharlotteJu** [reply]
It's perfect !! Thank you so much
